pyvista/core/utilities/geometric_sources.py

```diff
@@ -609,12 +609,12 @@ def __init__(
         self.capping = capping
 
     @property
-    def center(self) -> Sequence[float]:
+    def center(self) -> tuple[float, float, float]:
```

Comment:
For this to be true, I think we need to convert value within the setter, i.e. `self._center = tuple(center)`?

Reply:
Good catch, this is needed for Text3DSource too, should be fixed with https://github.com/pyvista/pyvista/pull/6469/commits/57d0c8e48a1e1528d280ad3fe2e7401bebd26c6f